plugins/nodes/src/js/pages/__tests__/NodeDetailPage-test.js

```diff
@@ -1,123 +0,0 @@
-const JestUtil = require("#SRC/js/utils/JestUtil");
```

Comment:
I found this test to be really hard to migrate and not really of value because the only check was `expect(node instanceof Node).toEqual(true);`

Reply:
Why do you think it provides value?